### DISCUSSION TITLE
Wazuh-agent on LXC Container running on Proxmox

### DISCUSSION THREAD

**Discussion by dauphinpasdroit** [wazuh/wazuh#24954] [Q&A]
Hello,

I would like to use Wazuh to monitor the security health of some server I run on a proxmox server.

It seems like I can't install the agent on LXC container. Here is the message error: 

```
2024/07/27 17:01:15 wazuh-agentd: ERROR: (4112): Invalid server address found: 'MANAGER_IP'
```

I found out that I needed to check in the `/var/ossec/etc/ossec.conf` but it seems like this file doesn't exist in an LXC Container. 

Here is the OS information:

/etc/os-release
```
PRETTY_NAME="Debian GNU/Linux 12 (bookworm)"
NAME="Debian GNU/Linux"
VERSION_ID="12"
VERSION="12 (bookworm)"
VERSION_CODENAME=bookworm
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

hostnamectl 
```
 Static hostname: ldap
       Icon name: computer-container
         Chassis: container ☐
      Machine ID: 14051b1b3cb14d209775944e3a05426a
         Boot ID: 4ef7f86d50ec4f9895609d7ffc7de264
  Virtualization: lxc
Operating System: Debian GNU/Linux 12 (bookworm)
          Kernel: Linux 6.5.13-1-pve
    Architecture: x86-64
```

Do you have any ideas of what I could do to make it work on LXC Containers ?

**Comment by askthedads** [comment]
@dauphinpasdroit this Happened to me recently. I needed to edit /var/ossec/etc/ossec.conf due to an error during install I had to correct first. 

You will see at the top in the `<server>` section under the address field field, it likely looks like this: `<address>MANAGER_IP</address>`. 

Replace that with the manager IP and do a daemon-reload, enable wazuh-agent and start.

I would completely purge the install using apt remove --purge wazuh-agent.
